templates/themes/app/pagelayout/header/language_selector.html.twig

```diff
@@ -2,9 +2,9 @@
     {% set prioritized_languages = ibexa.configResolver.getParameter('languages') %}
 
     <div class="language-selector dropdown">
-        <a href="#lang-selector" data-bs-toggle="dropdown" class="current-lang">
+        <button data-bs-toggle="dropdown" class="current-lang">
```

Comment:
@TTrakostanec  isto kao i za mainnav-toggle:

aria-expanded="false" (promijeniti na klik na true)
aria-controls="lang-selector" -> samo što se to referencira na id, a ne klasu pa treba i na taj element dodati id
aria-label="Change language"

Reply:
fixed in https://github.com/netgen/media-site/pull/214/commits/f1fbfc5dc05bb50aef14f755dd47c099f632e6b7